backend/Makefile

```
@@ -0,0 +1,100 @@
+# Makefile for SEPE Templates Comparator Backend
+
+.PHONY: help install install-dev test test-cov lint format type-check clean run dev migrate upgrade downgrade shell
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)